src/powerhouse/components/drag-and-drop/draggable.tsx

```diff
@@ -0,0 +1,32 @@
+import { useDrag } from 'react-aria';
+import { CUSTOM_OBJECT_FORMAT } from './constants';
+
+export interface DraggableRenderProps {
+    isDragging: boolean;
+}
+
+export interface DraggableProps<Item = any>
+    extends Omit<React.HTMLAttributes<HTMLDivElement>, 'children'> {
+    children?: (props: DraggableRenderProps) => React.ReactNode;
+    item: Item;
+}
+
+export function Draggable<Item = any>(props: DraggableProps<Item>) {
+    const { item, children, ...divProps } = props;
+
+    if (!children) return null;
+
+    const { dragProps, isDragging } = useDrag({
+        getItems: () => [
+            {
+                [CUSTOM_OBJECT_FORMAT]: JSON.stringify(item),
```

Comment:
This should be configurable, no? So that there can be multiple drag and drop components in the same page and the items can easily be distinguished.
For instance, in Connect there are the tabs and the tree view.